code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/vip_extraction.dm

```diff
@@ -134,3 +147,92 @@
 	head = /obj/item/clothing/head/helmet
 	l_hand = /obj/item/melee/baton/loaded
 	r_hand = /obj/item/gps
+
+//=====================
+// Marooned Syndicate Operative
+//=====================
+
+/datum/antagonist/marooned_syndicate_op
+	name = "Marooned Syndicate Operative"
+	show_in_antagpanel = TRUE
+	roundend_category = "Ruin VIPs"
+	antagpanel_category = "Other"
+/datum/outfit/syndicate_op
+	name = "Marooned Syndicate Operative"
+
+	uniform = /obj/item/clothing/under/syndicate/combat
+	suit = /obj/item/clothing/suit/space/syndicate
+	shoes = /obj/item/clothing/shoes/combat
+	gloves = /obj/item/clothing/gloves/combat
+	ears = /obj/item/radio/headset/syndicate/alt
```

Comment:
Don't give a syndie headset since it will screw up all traitors considering the ENTIRE syndicate comms is on a single channel